uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -756,15 +794,66 @@ public function upgrade_1033() {
   }
 
   /**
-   * Renames the case status Ongoing and Resolved to In Progress and Completed,
+   * Renames the case status Ongoing and Resolved to In Progress and Completed.
    *
    * @return bool
    */
   public function upgrade_1034() {
     $this->_relabelCaseStatus('Ongoing', 'In Progress');
     $this->_relabelCaseStatus('Resolved', 'Completed');
+
+    return TRUE;
   }
 
+  /**
+   * Deletes activity types that are not needed by T&W.
```

Comment:
T&W?

Reply:
Tasks and Workflows. That's the name of the extension. Should I change it to something else?

Reply:
Ah, I'd never heard of T&W, apparently it's a thing and I'm just stuck in the past. Never mind my comment